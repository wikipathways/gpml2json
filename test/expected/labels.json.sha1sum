15b05ff6bd808f46efc7894579381e2aa5ee042a ?-
