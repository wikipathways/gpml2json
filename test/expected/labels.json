{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["a14a0","bd539","f4e58","ac5bb","f3bc5","a2cca","fb9a8","dab04","fcadd","ff6ef","da678","f83ef","e7b2b","cbd2a"],"drawAs":"rect","gpmlElementName":"Pathway","height":185.993665670104,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":1442.3016983016983,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/3f760a8c7bfe6827d0d4fc4216d35db4e790cf231f7cc61145b46febeaa03dce"},"entityMap":{"a14a0":{"id":"a14a0","textContent":"Label\nfont:  Times New Roman","x":47.279798240527754,"y":69.17569686439079,"backgroundColor":"#99ffff","fontFamily":"Times New Roman","fontSize":15,"height":86.8179688057132,"drawAs":"rect","verticalAlign":"middle","width":176.21451268746466,"zIndex":32832,"textAlign":"center","color":"#000000","textDecoration":"none","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"},"bd539":{"id":"bd539","textContent":"Label","x":289.3300451978405,"y":68.47028504632709,"backgroundColor":"#99ffff","fontSize":10,"height":20,"drawAs":"rect","verticalAlign":"middle","width":80,"zIndex":32833,"textAlign":"center","color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"},"f4e58":{"id":"f4e58","textContent":"Label","x":324.5465341408019,"y":118.97339819311014,"backgroundColor":"#99ffff","fontSize":10,"height":20,"drawAs":"rect","verticalAlign":"middle","width":80,"zIndex":32863,"textAlign":"center","color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"},"ac5bb":{"id":"ac5bb","textContent":"Label","x":295.5083891468851,"y":105.22187740301501,"fontSize":10,"height":20,"drawAs":"rect","verticalAlign":"middle","width":80,"zIndex":32864,"textAlign":"center","color":"#000000","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"},"f3bc5":{"id":"f3bc5","textContent":"Label","x":454.744057936694,"y":65.73143154314242,"backgroundColor":"#99ffff","fontSize":10,"fontWeight":"bold","height":20,"drawAs":"rect","verticalAlign":"middle","width":80,"zIndex":32865,"textAlign":"center","color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"},"a2cca":{"id":"a2cca","textContent":"Label","x":458.3746311851016,"y":110.4958360252928,"backgroundColor":"#99ffff","fontSize":10,"fontStyle":"italic","height":20,"drawAs":"rect","verticalAlign":"middle","width":80,"zIndex":32866,"textAlign":"center","color":"#000000","textDecoration":"none","fontFamily":"Arial","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"},"fb9a8":{"id":"fb9a8","textContent":"Label","x":570.2854592105792,"y":70.44480733932077,"color":"#ff0000","backgroundColor":"#99ffff","fontSize":10,"fontStyle":"italic","height":20,"drawAs":"rect","verticalAlign":"middle","width":80,"zIndex":32867,"textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"},"dab04":{"id":"dab04","textContent":"Label","x":572.6421471086685,"y":116.11359714823796,"color":"#ff0000","backgroundColor":"#99ffff","fontSize":10,"height":20,"drawAs":"Ellipse","verticalAlign":"middle","width":80,"zIndex":32868,"textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"},"fcadd":{"id":"fcadd","textContent":"Label","x":687.8650770449742,"y":115.41296020556277,"backgroundColor":"#99ffff","fontSize":20,"height":37.45328183265377,"drawAs":"rect","verticalAlign":"middle","width":93.02664270024869,"zIndex":32870,"textAlign":"center","color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"},"ff6ef":{"id":"ff6ef","textContent":"Label","x":839.3268247011688,"y":71.62921348314609,"backgroundColor":"#99ffff","fontSize":10,"height":27.363507689997107,"borderWidth":5,"drawAs":"rect","verticalAlign":"middle","width":86.23497305164017,"zIndex":32871,"textAlign":"center","color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"},"da678":{"id":"da678","href":"http://www.pathvisio.org","textContent":"Label","x":841.8141216309621,"y":124.13907485524436,"color":"#0000ff","backgroundColor":"#99ffff","fontSize":10,"height":20,"drawAs":"rect","verticalAlign":"middle","width":80,"zIndex":32872,"textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"},"f83ef":{"id":"f83ef","textContent":"Label","x":699.3300451978403,"y":65.46215496502629,"backgroundColor":"#99ffff","fontSize":10,"height":20,"strokeDasharray":"5,3","drawAs":"rect","verticalAlign":"middle","width":80,"zIndex":32873,"textAlign":"center","color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"},"e7b2b":{"id":"e7b2b","textContent":"SAVE","x":164.8151848151848,"y":14.995004995005,"backgroundColor":"transparent","fontSize":30,"fontWeight":"bold","height":27.982017982017986,"verticalAlign":"middle","width":95.94405594405595,"zIndex":33015,"textAlign":"center","color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","borderWidth":0,"drawAs":"none","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"},"cbd2a":{"id":"cbd2a","textContent":"This pathway is to be saved long-term to test the ability of\nGPML renderers to handle the element type named in the title.","textAlign":"left","x":268.7112887112888,"y":9.000999000999006,"backgroundColor":"transparent","fontSize":10,"height":32.977022977022976,"width":290.7492507492508,"zIndex":33016,"color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":0,"drawAs":"none","verticalAlign":"top","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"}}}
